src/main/java/frc/robot/subsystems/CoralElevator.java

```diff
@@ -0,0 +1,53 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotMap;
+
+public class CoralElevator extends SubsystemBase {
+
+    private DoubleSolenoid piston1;
+    private DoubleSolenoid piston2;
```

Comment:
```suggestion
    private final DoubleSolenoid piston2;
```
finalize when possible